src/components/tutorials-sidebar/types.ts

```diff
@@ -39,8 +41,13 @@ export interface ListItemProps {
 	text: string
 }
 
+export interface TutorialListItemProps extends ListItemProps {
+	tutorialId: uuid
+	collectionId: uuid
+}
```

Comment:
Should `Tutorial['id']` and `Collection['id']` be referenced here? Those should probably reference `uuid` as their source of truth if they're not. 💭

Reply:
✅ ah great call, updated!